projects/managed-hubs/support.md

```diff
@@ -1,118 +1,236 @@
+1. Requests that have a high-impact on a single community.
+1. Our other work items
+1. Requests from communities that have no more {term}`Support Budget`.
+
+:::{admonition} Ask the PMs
+We have a few roles that are particularly useful for triaging and prioritizing our work items.
+
+- Our {term}`Project Manager` should have visibility over everybody's workload, and helps us prioritize work during sprints.
+- Our Product and Community Lead should understand which work items will be most impactful across our communities. (we currently do not have anybody serving in this role, but will soon!)
+
+When deciding how to prioritize a Change Request, ask for guidance from these two team members.
+
+:::
 
-- Ask questions that will help us understand whether an issue is "critical". For example, "Does this effect many users on the hub, or just a subset?", "Does this problem always happen, or intermittently?", "Is the hub accessible at all or is it totally down?".
-- Events should be prioritized (or in general, moments with a large influx of users)
-- In general, "how much does this incident impact the users of a given hub?" is a good guideline for deciding what to prioritize
+### Support request key terms
 
+The following are other important terms and ideas in the support process.
 
-## Asking for help from others
+```{glossary}
+Support Budget
+  A fixed amount of time that we can spend providing support for each community that we serve. This helps us ensure that we can sustainably serve many communities. Any support request that is **not tied to an {term}`Incident`** will draw from the support budget for that community. If a community requests support beyond their support budget, we may request extra funds to help cover our costs.
 
-The support team is primarily a **communicator** and a **router** - they are not necessarily the ones who carry out the changes needed to resolve a support issue.
-Instead, the support team is empowered to ask engineering team members to pick up backlog items that are related to support requests.
+  :::{note}
+  We currently keep this term intentionally vague, and ask that communities are respectful of our time when making change requests.
+  We are investigating the support budget that we should give to each community, and will update here when we have specific numbers in mind. 
 
-When a support request is made that requires an action from a 2i2c engineer, a Support Steward should describe this change in a GitHub issue, and add it to the [Sprint Board](coordination:sprint-board).
-Think about an engineering team member that likely has the skills and capacity needed, and ask them if they are willing to take on resolving this issue.
-Try not to ask the same person for support help many times in a row - we should spread the work needed to address support issues across the team.
+  Here is a rough idea of our rationale to follow for arriving at a specific number:
```

Comment:
You mention arriving at "a specific number" but I think you did not mention that specific number above, is that intended?

From your previous comment, that number would be:

> we could say that those various hub types correspond to 34/20 ~= 1 hours, or 34 / 8 ~= 4 hours of support each month.

I know you can derive that from the rationale you have included but I feel it needs a conclusion like the above one.

Reply:
I'll try to make it clear that this number is not yet specific, and that we want to keep it as a rationale only, but no precise numbers.